src/System.Runtime.Numerics/tests/BigInteger/add.cs

```diff
@@ -1,160 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using Xunit;
-
-namespace System.Numerics.Tests
-{
-    public class addTest
-    {
-        private static int s_samples = 10;
-        private static Random s_random = new Random(100);
-
-        [Fact]
-        public static void RunAddTests()
-        {
-            byte[] byteArray1 = new byte[0];
-            byte[] byteArray2 = new byte[0];
-
-            // Add Method - Two Large BigIntegers
-            for (int i = 0; i < s_samples; i++)
-            {
-                byteArray1 = GetRandomByteArray(s_random);
-                byteArray2 = GetRandomByteArray(s_random);
```

Comment:
It seems like this is currently providing more coverage, in that it's testing a wider array of inputs.  Could you run this code, get the "random" big integers it generates, and add those 10 inputs to your member data, so that we're not losing anything here?  You can add a comment that, in addition to the more structure examples (like min and max) these are randomly generated.  (Same comment applies to other cases of this.)